tests/run_tests.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Exit status used to report back to caller
+#
+STATUS=0
+
+# Directory containing this script, used to ensure artefacts are placed in the
+# mounted volumes.
+#
+TEST_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &> /dev/null && pwd)
+
+export HDF5_DISABLE_VERSION_CHECK=1
+
+
+# Run the standard set of unit tests, producing JUnit compatible output
+#
+pytest -s --cov=gdal_subsetter --junitxml=${TEST_DIR}/reports/hga_junit.xml --cov-report=html:${TEST_DIR}/coverage
```

Comment:
I really like the `pytest` plugin ecosystem. It's one of the things I miss about it compared to `unittest`, which we use instead for a lot of our other Data Services projects.